specification/ilm/_types/Phase.ts

```diff
@@ -25,10 +25,7 @@ import { Duration, DurationLarge } from '@_types/Time'
 
 export class Phase {
   actions?: Actions
-  /**
-   * @es_quirk output as a millis number in XPack usage stats, which cannot roundtrip with a Duration as it requires a unit.
-   */
-  min_age?: Duration | long
+  min_age?: Duration
```

Comment:
It turns out ILM and Xpack usage output different representations for `min_age`. It's always a TimeValue, but:

 * ILM APIs [call getStringRep()](https://github.com/elastic/elasticsearch/blob/2e9ff9ae66bff80df6e8a2149fdef818a6efe573/x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/Phase.java#L158C51-L158C61) which outputs [units for all non-negative values](https://github.com/elastic/elasticsearch/blob/6bb0799893f4ee4e6afa7094346fbc51d5203538/libs/core/src/main/java/org/elasticsearch/core/TimeValue.java#L341-L354)
 * XPack usage [calls getMillis()](https://github.com/elastic/elasticsearch/blob/b4e852a54be436d8b8036da0a0ec4a472d44524a/x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/IndexLifecycleFeatureSetUsage.java#L189C61-L189C71) which [returns a long](https://github.com/elastic/elasticsearch/blob/b4e852a54be436d8b8036da0a0ec4a472d44524a/libs/core/src/main/java/org/elasticsearch/core/TimeValue.java#L133-L135) which is why I'm using `DurationValue<UnitMillis>` now.